### PR TITLE
Initial tutorial

### PR DESCRIPTION
**Pull request by idomic**
This covers the initial tutorial for underfitting and overfitting, how a learning curve can solve it and some examples.

### CODE REVIEW

**Comment by idomic** [comment]
That's only a first draft.

We can also shrink the texts and add more code.

This [example for instance](https://scikit-learn.org/stable/auto_examples/model_selection/plot_underfitting_overfitting.html), is good, but is lacking context and background.

File: docs/tutorials/underfitting_overfitting.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How to deal with underfitting and overfitting?
```
Comment (edublancas) [inline docs/tutorials/underfitting_overfitting.md:14]: the content is good but I think we can re-organize.

first, quick intro to the concepts

then, tell the user how to diagnose. see if there's underfitting/overfitting/none

then one section for tackling overfitting and another one for underfitting

I think adding some external references can help as well us enrich the content, and help with SEO

File: docs/tutorials/underfitting_overfitting.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+### Underfitting:
+Increasing the model's complexity, Adding/Collecting more data, Using more informative features, ensemble methods, hyperparameter tuning, transfer learning.
+
+
++++
+
+# Conclusion
+
+```{code-cell} ipython3
```
Comment (edublancas) [inline docs/tutorials/underfitting_overfitting.md:84]: looks like from here til the bottom it was copied from another file?

**Review by edublancas — CHANGES_REQUESTED** [review]

File: docs/tutorials/underfitting_overfitting.md (hunk 2 above)
Comment (idomic) [reply docs/tutorials/underfitting_overfitting.md:84]: Yup, I thought I removed it.

**Comment by idomic** [comment]
Closing this.

**Pull request closed without merging**